cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -50,25 +52,37 @@ func CreateLocalEtcdStaticPodManifestFile(manifestDir string, cfg *kubeadmapi.Ma
 func GetEtcdPodSpec(cfg *kubeadmapi.MasterConfiguration) v1.Pod {
 	pathType := v1.HostPathDirectoryOrCreate
 	etcdMounts := map[string]v1.Volume{
-		etcdVolumeName: staticpodutil.NewVolume(etcdVolumeName, cfg.Etcd.DataDir, &pathType),
+		etcdVolumeName:  staticpodutil.NewVolume(etcdVolumeName, cfg.Etcd.DataDir, &pathType),
+		certsVolumeName: staticpodutil.NewVolume(certsVolumeName, cfg.CertificatesDir, &pathType),
```

Comment:
Should this just mount in the etcd certs here, instead of all of the certs?

Reply:
This will be easy for this PR if we have a separate etcd CA cert in the same subdirectory directory.

I think we should do this for all of the static pods, though.
It's a little weirder when we think about the apiserver pod needing the etcd CA cert.

This seems doable with individual HostPath Volumes per File -- WDYT?

Reply:
I think it all depends on the end-user experience when managing the certs post-install. I'm a bit torn on whether referencing single files across different paths makes the most sense here or if duplicating files into component based subdirectories makes the most sense.